src/System.Management.Automation/namespaces/RegistryProvider.cs

```diff
@@ -3061,7 +3061,7 @@ private bool HasRelativePathTokens(string path)
 
                     if (
                         expandAll ||
-                        ((Context.SuppressWildcardExpansion == false) && (valueNameMatcher.IsMatch(valueNameToMatch))) ||
+                        ((!Context.SuppressWildcardExpansion) && (valueNameMatcher.IsMatch(valueNameToMatch))) ||
                        ((Context.SuppressWildcardExpansion) && (string.Equals(valueNameToMatch, requestedValueName, StringComparison.OrdinalIgnoreCase))))
```

Comment:
Extra parentheses.

Reply:
I will fix this in seperate PR using [RCS1032](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1032.md): Remove redundant parentheses

Reply:
Actually neither RCS1032 nor IDE0047 classify these parentheses as unecessary.

Reply:
Anyway I would prefer remove parentheses in seperate PR.